MekHQ/src/mekhq/campaign/stratcon/StratconCampaignState.java

```diff
@@ -176,6 +176,20 @@ public void useSupportPoint() {
         supportPoints--;
     }
 
+    /**
+     * Decreases the number of support points by the specified increment.
+     *
+     * @param increment The number of support points to use/decrease.
+     */
+    public void useSupportPoints(int increment) {
+        supportPoints -= increment;
+    }
```

Comment:
I guess thats a decrement

Reply:
I do believe you would be correct, yes :D